hs_sitemap/sitemaps.py

```diff
@@ -11,3 +13,17 @@ def items(self):
 
     def location(self, item):
         return reverse(item)
+
+
+class ResourcesSitemap(sitemaps.Sitemap):
+    priority = 0.5
+    changefreq = 'daily'
+
+    def items(self):
+        return BaseResource.objects.filter(Q(raccess__public=True) | Q(raccess__discoverable=True))
+
+    def location(self, item):
+        return item.get_absolute_url()
```

Comment:
Better to use the property `absolute_url `(no db query with this property)

Reply:
good call thanks. updated.